apcd-cms/src/apps/admin_extension/views.py

```diff
@@ -84,7 +64,7 @@ def getDate(row):
             context['extensions'].append(self._set_extension(extension))
 
             entity_name = title_case(extension[18])
```

Comment:
Should these checks be added here? Not sure if add `and not "None"` will add any regressions to the way we add options to filters.


```       
entity_name = title_case(extension[18]) if extension[18] else "None"
status = title_case(extension[7]) if extension[7] else "None"
outcome = title_case(extension[8]) if extension[8] else "None"
if entity_name not in context['org_options'] and not "None":
   context['org_options'].append(entity_name)
   context['org_options'] = sorted(context['org_options'], key=lambda x: (x != 'All', x))
   # Remove empty strings
    context['org_options'] = [option for option in context['org_options'] if option != '']
if status not in context['status_options'] and not "None":
     context['status_options'].append(status)
     context['status_options'] = sorted(context['status_options'], key=lambda x: (x != 'All', x))
if outcome not in context['outcome_options'] and not "None":
    context['outcome_options'].append(outcome)
    context['outcome_options'] = sorted(context['outcome_options'], key=lambda x: (x is not None, x))```